app/src/main/java/openfoodfacts/github/scrachx/openfood/views/LoginActivity.java

```diff
@@ -154,6 +154,11 @@ protected void attemptLogin() {
             loginView.requestFocus();
             return;
         }
+        if (TextUtils.isEmpty(password)) {
+            Toast.makeText(this, getString(R.string.error_field_required), Toast.LENGTH_SHORT).show();
```

Comment:
Can you please set this as an error on the password view, rather than a toast, just to make sure that it is kept consistent. Check out a couple of lines below.

Reply:
@Karljoones  I used toast because to keep consistent as I refer to the PR I made in past i.e..https://github.com/openfoodfacts/openfoodfacts-androidapp/pull/2295

Reply:
Text field error messages should be in the password view as described by @Karljoones . In the login activity, we are using `TextInputLayout`. The reason the icons are overlapping is because we should actually be setting the error message to this layout and not the Edit Text view. 

See here for more information: https://material.io/develop/android/components/text-input-layout/

Reply:
@Karljoones I have made the required changes,Check it out once :)